src/cms/templates/push_notifications/push_notification_form.html

```diff
@@ -7,6 +7,11 @@
 {% load page_filters %}
 {% load rules %}
 
+<head>
+<script src="{% static 'charCounter.js' %}"></script>
+</head>
+
+
```

Comment:
```suggestion
```
No need to include static files in the HTML templates - we use Webpack as bundling tool, so just add your script to [src/frontend/index.ts](https://github.com/Integreat/integreat-cms/blob/develop/src/frontend/index.ts) and everything will be included automatically.